modules/distribution-zones/src/test/java/org/apache/ignite/internal/distributionzones/BaseDistributionZoneManagerTest.java

```diff
@@ -59,6 +61,18 @@
  */
 @ExtendWith(ConfigurationExtension.class)
 public class BaseDistributionZoneManagerTest extends BaseIgniteAbstractTest {
+    static final String ZONE_NAME_0 = "zone0";
+
+    static final String ZONE_NAME_1 = "zone1";
+
+    static final String ZONE_NAME_2 = "zone2";
+
+    static final LogicalNode NODE_0 = new LogicalNode("node0", "node0", new NetworkAddress("localhost", 123));
```

Comment:
It's better to use different node name and id, just in case.

Reply:
Fixed